diff --git a/node_modules/h3-js/dist/browser/h3-js.es.js b/node_modules/h3-js/dist/browser/h3-js.es.js
index 5f74d6f..7cdd692 100644
--- a/node_modules/h3-js/dist/browser/h3-js.es.js
+++ b/node_modules/h3-js/dist/browser/h3-js.es.js
@@ -24,9 +24,9 @@ var libh3 = function (libh3) {
   var readAsync;
 
   {
-    if (document.currentScript) {
-      scriptDirectory = document.currentScript.src;
-    }
+    // if (document.currentScript) {
+    //   scriptDirectory = document.currentScript.src;
+    // }
 
     if (scriptDirectory.indexOf("blob:") !== 0) {
       scriptDirectory = scriptDirectory.substr(0, scriptDirectory.lastIndexOf("/") + 1);
diff --git a/node_modules/h3-js/dist/browser/h3-js.js b/node_modules/h3-js/dist/browser/h3-js.js
index be0058c..451d04a 100644
--- a/node_modules/h3-js/dist/browser/h3-js.js
+++ b/node_modules/h3-js/dist/browser/h3-js.js
@@ -24,9 +24,9 @@ var libh3 = function (libh3) {
   var readAsync;
 
   {
-    if (document.currentScript) {
-      scriptDirectory = document.currentScript.src;
-    }
+    // if (document.currentScript) {
+    //   scriptDirectory = document.currentScript.src;
+    // }
 
     if (scriptDirectory.indexOf("blob:") !== 0) {
       scriptDirectory = scriptDirectory.substr(0, scriptDirectory.lastIndexOf("/") + 1);
